src/components/docs.jsx

```diff
@@ -25,10 +25,51 @@ class Docs extends React.Component {
       }
     };
   }
+  getVideoStyles() {
+    return {
+      grid: {
+        [settings.mediaQueries.medium]: {
+          paddingLeft: "5%",
+          paddingRight: "5%"
+        },
+        [settings.mediaQueries.large]: {
+          paddingLeft: "10%",
+          paddingRight: "10%"
+        }
+      },
+      wrapper: {
+        position: "relative",
+        paddingBottom: "56.25%", // 16:9 ratio
+        paddingTop: "25px",
+        height: 0
+      },
+      iframe: {
+        position: "absolute",
+        top: 0,
+        left: 0,
+        width: "100%",
+        height: "100%"
+      }
+    };
+  }
   render() {
     const spectacleDocs = marked(SpectacleREADME);
+    const videoStyles = this.getVideoStyles();
     return (
       <section style={this.getSectionStyles()}>
+        <div style={videoStyles.grid}>
+          <div style={videoStyles.wrapper}>
+            <iframe
```

Comment:
I ticketed this out: https://github.com/FormidableLabs/formidable-landers/issues/76